module/VuFind/src/VuFind/Autocomplete/Solr.php

```diff
@@ -360,6 +376,19 @@ protected function setSortField($new)
         $this->sortField = $new;
     }
 
+    /**
+     * Set the limit. Useful for child classes.
+     *
+     * @param int $limit Limit of search result rows. Can be null to default
```

Comment:
We should make this match the actual type:

```suggestion
     * @param ?int $limit Limit of search result rows. Can be null to default
```

Reply:
Since this is a trivial change, I'll just go ahead and apply it now. :-)